src/masternodes/govvariables/attributes.cpp

```diff
@@ -1611,6 +1653,14 @@ Res ATTRIBUTES::Validate(const CCustomCSView &view) const {
                 }
                 break;
 
+            case AttributeTypes::Vaults:
+                if (attrV0->typeId == VaultIDs::DUSDVault && attrV0->key == VaultKeys::DUSDVaultEnabled) {
+                    if (view.GetLastHeight() < Params().GetConsensus().NextNetworkUpgradeHeight) {
+                        return Res::Err("Cannot be set before NextNetworkUpgrade");
```

Comment:
In a separate PR, can we pull all of these NextNetworkUpgrade strings, and collect it under DeFiErrors? Once we name the upgrade, should also help to just make a single change that chase after all these strings.